packages/now-cli/src/util/dev/builder.ts

```diff
@@ -64,6 +66,7 @@ async function createBuildProcess(
   let PATH = `${dirname(process.execPath)}${delimiter}${process.env.PATH}`;
 
   const env: Env = {
+    ...systemEnvs.buildEnv,
     ...process.env,
```

Comment:
Do we want the local process.env to overwrite system env vars? Seems like it should be the other way around.

Reply:
Agreed, I changed the order 👍